Include/internal/pycore_gc.h

```diff
@@ -329,6 +334,7 @@ struct _gc_runtime_state {
     Py_ssize_t work_to_do;
     /* Which of the old spaces is the visited space */
     int visited_space;
+    int phase;
```

Comment:
```suggestion
    enum _GCPhase phase;
```

Reply:
Unfortunately the size of `enum` is not defined (blame the C standards committee), so we avoid using `enum` in structs.
We'll have to wait until C23 for sized enums https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3030.htm